pkg/scheduler/plugins/reservation/plugin.go

```diff
@@ -236,12 +237,14 @@ func (pl *Plugin) PostFilter(ctx context.Context, cycleState *framework.CycleSta
 		}
 		reservationInfos := pl.reservationCache.listReservationInfosOnNode(node.Name)
 		for _, rInfo := range reservationInfos {
-			newReservePod := reservationutil.NewReservePod(rInfo.Reservation)
```

Comment:
In the original implementation, the data that has been corrected by NodeInfo may be added back again